### PR TITLE
New Security Panel blog post

### PR DESCRIPTION
**Pull request by pbakaus**
New Security Panel blog post on the new DevTools panel. Ready for review.

/cc @kaycebasques

### CODE REVIEW

**Comment by PaulKinlan** [comment]
Looks good.  Has Emily reviewed?

File: src/content/en/updates/posts/2015/12/security-panel.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+tags:
+  - devtools
+  - update
+description: "Learn all about the new Security Panel in Chrome DevTools."
+featured_image: /web/updates/images/2015/12/security-panel/security-panel.png
+---
+
+![The new Security Panel](/web/updates/images/2015/12/security-panel/security-panel.png)
+
+The Chrome Security team has been [hard at work](https://www.chromium.org/Home/chromium-security/marking-http-as-non-secure) to realize a future without HTTP, a future where you and your users can be reasonably sure that whatever data you’re sending to the web stays between you and the site you’re looking at. And to to make it even easier to jump ship and join the glorious HTTPS future, we’ve made Security a first-class citizen in the DevTools.
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2015/12/security-panel.markdown]: s/in the DevTools/in DevTools/

File: src/content/en/updates/posts/2015/12/security-panel.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+ - It’s too complicated for most users
+ - ...but too basic for most developers
+ - and makes it unclear what causes a lock icon “downgrade”
+
+## Overview: Explain lock icon and surface mixed content
+
+![Overview tab](/web/updates/images/2015/12/security-panel/overview.png)
+
+The **lock icon** represents the security state of the page, so knowing when and why it appears is extremely important. The overview screen in the new security panel explains the important parts that contribute to the lock icon:
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2015/12/security-panel.markdown]: s/contribute to the lock icon/contribute to a secure page/

File: src/content/en/updates/posts/2015/12/security-panel.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+ - **Identity** (certificate)
+ - **Connection** (protocol, cipher suite)
+ - **Subresources**
+
+You’ll now know at a glance why your site does or does not get the little green badge of awesomeness.
+
+Have **mixed content** appear out of nowhere? No worries. We show it directly on the overview, and a click brings you to as filtered view of the Network Panel, so you can quickly look at the offending requests:
```
Comment (kaycebasques) [inline src/content/en/updates/posts/2015/12/security-panel.markdown]: s/to as filtered view/to a filtered view/

**Comment by kaycebasques** [comment]
:+1: aside from some minor edits

**Comment by petele** [comment]
I plan to do a push today around 11am PT, if you want this in for then, LMK it's good to go.

File: src/content/en/updates/posts/2015/12/security-panel.markdown (hunk 1 above)
Comment (pbakaus) [inline src/content/en/updates/posts/2015/12/security-panel.markdown]: This came directly via @paulirish, so I think it might be best to leave it :)

Comment (pbakaus) [inline src/content/en/updates/posts/2015/12/security-panel.markdown]: oh nevermind, thought you meant it the other way around. Will fix.

**Comment by pbakaus** [comment]
@petele @kaycebasques edits done, post is good to go. Feel free to merge!

@PaulKinlan not sure who Emily is, but if you're referring to Elisabeth, then yes :) @paulirish reviewed as well.

**Comment by pbakaus** [comment]
@PaulKinlan ah, that Emily! Found her. She did not review it, but as Elisabeth did, I think we're good to go.

**Comment by PaulKinlan** [comment]
Hah, yes Emily stark.

**Pull request merged**